src/materialized/src/bin/materialized/main.rs

```diff
@@ -362,6 +362,9 @@ pub struct Args {
     )]
     opentelemetry_headers: Option<String>,
 
+    #[clap(long, env = "MZ_ALLOW_CLUSTER_REPLICA_SIZE")]
+    allow_cluster_replica_size: Option<String>,
```

Comment:
Nit, but the issue calls for a `cluster_replica_size` argument.

Reply:
Oh, but the design doc calls for it with `allow`: https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/20220413_cluster_replica.md#controller-changes

I think it doesn't really matter; I'll change it to `--cluster-replica-sizes` to be shorter (and plural, which makes more logical sense)

Reply:
done